src/Command/Argument/Hash/CommonExpiration.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) 2009-2020 Daniele Alessandri
+ * (c) 2021-2023 Till Krüss
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Predis\Command\Argument\Hash;
+
+use Predis\Command\Argument\ArrayableArgument;
+use UnexpectedValueException;
+
+abstract class CommonExpiration implements ArrayableArgument
+{
+    /**
+     * @var array
+     */
+    protected $expireModifierEnum = [
```

Comment:
In my mind these two don't belong together. The {NX,XX,GT,LT} set is for overwriting rules, while {EX,PX,EXAT,PXAT} are basically just parameter name in compact form. For example {NX,XX,GT,LT} applies to other hash expiration commands, while the other set does not.

Reply:
So something like `overrideModifierEnum` sounds better?